lib/core/tests/html_renderer.rs

```diff
@@ -187,24 +191,24 @@ mod tests {
 
     #[web_bench]
     fn object_x1000(b: &mut Bencher) {
-        let mut scene : Scene<HTMLObject> = Scene::new();
+        let logger = Logger::new("object_x1000");
+        let mut scene = HTMLScene::new(logger.clone());
         let renderer = HTMLRenderer::new("object_x1000")
                                     .expect("Renderer couldn't be created");
-        renderer.container.dom.set_property_or_panic("background-color", "black");
+        renderer.container().dom.set_property_or_panic("background-color", "black");
 
         for _ in 0..1000 {
-            let mut object = HTMLObject::new("div")
+            let mut object = HTMLObject::new(logger.clone(), "div")
                                     .expect("Failed to create object");
             object.set_dimensions(1.0, 1.0);
-            object.set_scale(0.5, 0.5, 0.5);
+            object.set_scale(Vector3::new(0.5, 0.5, 0.5));
             scene.add(object);
```

Comment:
The API should be consistent. `Scene` is display object, so adding children should be done by using `add_child`.

Reply:
As decided on Discord, this will be part of the World integration task.